### DISCUSSION TITLE
Repository DNS - IPv6 support

### DISCUSSION THREAD

**Discussion by inopinatus** [fullstaq-labs/fullstaq-ruby-server-edition#135] [Ideas]
**Feature request**

Support IPv6 for the official fullstaq package repository DNS names.

**Why**

I'm building an IPv6-only backend, which is entirely achievable in 2024, and one of the few wrinkles is that `apt.fullstaqruby.org` resolves only to an IPv4 address and has no IPv6 address.

**Workaround**

Observing that the repositories are currently implemented via a 302 redirect to Google storage buckets, I found this worked:

````
deb [signed-by=/etc/apt/trusted.gpg.d/fullstaq-ruby.gpg] https://storage.googleapis.com/fullstaq-ruby-server-edition-apt-repo/versions/24/public ubuntu-22.04 main
````

but this seems brittle, being neither documented nor recommended.

**Comment by FooBarWidget** [comment]
Looks like this is not easily possible with Google Kubernetes Engine's Ingress.

I've been thinking about switching back to a VPS, that'll allow me more control over this sort of stuff as well as save costs.
